.circleci/config.yml

```diff
@@ -116,6 +116,18 @@ references:
       command: |
         echo "Loading wiremock mappings..."
         find spec/wiremock/prison-api/mappings/*.json -exec curl --request POST --url http://localhost:8888/__admin/mappings/ --header 'content-type: application/json' --data-binary "@{}" \;
+  _notify_notify_sentry_release: &notify_notify_sentry_release
+    run:
+      name: Create release and notify Sentry of deploy
+      command: |
+        curl -sL https://sentry.io/get-cli/ | bash
+        export SENTRY_RELEASE=$(sentry-cli releases propose-version)
+        sentry-cli releases new -p $SENTRY_PROJECT $SENTRY_RELEASE
```

Comment:
I'm guessing SENTRY_PROJECT is coming from the circle ci context environment vars?

Reply:
Yes, a few SENTRY_ environment variables were defined in the project settings.
namely: 
```
    SENTRY_AUTH_TOKEN - Your internal integration token
    SENTRY_ORG - Your Sentry organization slug
    SENTRY_PROJECT - Your Sentry project slug
```